start-site/src/main/resources/application.yml

```diff
@@ -109,7 +109,7 @@ initializr:
         versionProperty: spring-cloud-azure.version
         mappings:
           - compatibilityRange: "[2.5.0.M1,3.0.0-M1)"
-            version: 4.3.0
+            version: 4.2.0
```

Comment:
Why are we no longer upgrading to 4.3.0?

Reply:
I created this [issue ](https://github.com/spring-io/start.spring.io/issues/955)to upgrade the bom version. Maybe it can be done with this pr？

Reply:
Given the title and description of this PR, I had expected it to include the upgrade to 4.3.0. It's completely fine if you'd prefer to separate the two, but the title and description of this PR should be updated. It looks it is now focussed solely on improving the Azure experience so you get better HELP.md contents, Azure-specific actuator extensions, etc.

Reply:
Updated, please take a look.